src/test/runner.ts

```diff
@@ -321,6 +322,22 @@ export class Runner {
   }
 }
 
+
+export function createSourceMapMatcher(patterns: string | RegExp | (string | RegExp)[]): Matcher {
+  const matcher = createMatcher(patterns);
+  return (filePath: string) => {
+    const sourceMap = retrieveSourceMap(filePath);
```

Comment:
I see that we provide custom `retrieveSourceMap` in `transform.ts`. Do they play nicely together?